django-stubs/db/models/fields/generated.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+from django.db import models
+from django.db.backends.base.base import BaseDatabaseWrapper
+from django.db.models.expressions import Expression
+from django.utils.datastructures import DictWrapper
+
+class GeneratedField(models.Field):
+    def __init__(
+        self, *, expression: Expression, output_field: models.Field, db_persist: bool | None = ..., **kwargs: Any
```

Comment:
I think it's better to add all the possible arguments, as it is done for other `Field` subclasses

Reply:
Done